apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -29,38 +27,41 @@ import {
   SaveType,
   ViewMode,
   Visibility,
+  ChatMessage,
+  ChatItem,
+  isModelUpdate,
+  isNotification,
+  isChatMessage,
+  ModelUpdate,
+  Notification,
 } from '../types';
 import {
-  AI_CUSTOMIZATIONS_LABELS,
   DEFAULT_VISIBILITIES,
   EMPTY_AI_CUSTOMIZATIONS,
 } from '../views/modelCustomization/constants';
 import {validateModelId} from '../views/modelCustomization/utils';
 
 import {
   allFieldsHidden,
-  decorateMessageFromModelResponse,
   findChangedProperties,
   getNewMessageId,
-  getCurrentTime,
-  getCurrentTimestamp,
   hasFilledOutModelCard,
 } from './utils';
 
-type MessageListKey = 'chatMessagesCurrent' | 'chatMessagesPast';
+const messageListKeys = ['chatItemsPast', 'chatItemsCurrent'] as const;
```

Comment:
Just curious - Is this the preferred way to create a type as opposed to what was initially here?

Reply:
Oh! I think it helps simplify these lines:
https://github.com/code-dot-org/code-dot-org/blob/33f65b5316207fbe4558a38b76d93c4a2bd1e243/apps/src/aichat/redux/aichatRedux.ts#L507-L508